holonix/app-spec/src/test.nix

```diff
@@ -0,0 +1,12 @@
+let
+  pkgs = import ../../nixpkgs/nixpkgs.nix;
+
+  name = "hc-app-spec-test";
+
+  script = pkgs.writeShellScriptBin name
+  ''
+  hc-conductor-node-install
```

Comment:
This could end up producing the hash mismatch error message (or failure) if run on it's own without a prior run of `hc-cli-install`

Reply:
@zippy sure, i think this has been problematic for a while so i don't want to change it here

two relevant items in SOA:

https://miro.com/app/board/o9J_kyiXmFs=/?moveToWidget=3074457346596892028

https://miro.com/app/board/o9J_kyiXmFs=/?moveToWidget=3074457346588039924